@xen-orchestra/lite/src/components/pool/network/PoolHostInternalNetworksTable.vue

```diff
@@ -0,0 +1,215 @@
+      />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import useMultiSelect from '@/composables/multi-select.composable'
+import { useNetworkStore } from '@/stores/xen-api/network.store'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import VtsStateHero from '@core/components/state-hero/VtsStateHero.vue'
+import ColumnTitle from '@core/components/table/ColumnTitle.vue'
+import VtsDataTable from '@core/components/table/VtsDataTable.vue'
+import UiButton from '@core/components/ui/button/UiButton.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTableActions from '@core/components/ui/table-actions/UiTableActions.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { useRouteQuery } from '@core/composables/route-query.composable'
+import { useTable } from '@core/composables/table.composable'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import {
+  faAlignLeft,
+  faCaretDown,
+  faEdit,
+  faEllipsis,
+  faHashtag,
+  faPlus,
+  faTrash,
+} from '@fortawesome/free-solid-svg-icons'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { networksWithoutPifs: networks, isReady, hasError } = useNetworkStore().subscribe()
+
+const { t } = useI18n()
+const searchQuery = ref('')
+const selectedNetworkId = useRouteQuery('id')
+
+const filteredNetworks = computed(() => {
+  const searchTerm = searchQuery.value.trim().toLocaleLowerCase()
+  if (!searchTerm) {
+    return networks.value
+  }
+  return networks.value.filter(network =>
+    Object.values(network).some(value => String(value).toLocaleLowerCase().includes(searchTerm))
+  )
+})
+
+const networkUuids = computed(() => networks.value.map(network => network.uuid))
+
+const { selected, areAllSelected } = useMultiSelect(networkUuids)
+
+const toggleSelect = () => {
+  selected.value = selected.value.length === 0 ? networkUuids.value : []
+}
+
+const getLockingMode = (lockingMode: string) => (lockingMode === 'disabled' ? t('disabled') : t('unlocked'))
+
+const { visibleColumns, rows } = useTable('networks', filteredNetworks, {
+  rowId: record => record.uuid,
+  columns: define => [
+    define('checkbox', () => '', { label: '', isHideable: false }),
+    define('name_label', { label: t('name') }),
+    define('name_description', { label: t('description') }),
+    define('MTU', { label: t('mtu') }),
+    define('default_locking_mode', record => getLockingMode(record.default_locking_mode), {
+      label: t('default-locking-mode'),
+    }),
+    define('more', () => '', { label: '', isHideable: false }),
+  ],
```

Comment:
We can use `noop` from `@vueuse/shared` here

```suggestion
  columns: define => [
    define('checkbox', noop, { label: '', isHideable: false }),
    define('name_label', { label: t('name') }),
    define('name_description', { label: t('description') }),
    define('MTU', { label: t('mtu') }),
    define('default_locking_mode', record => getLockingMode(record.default_locking_mode), {
      label: t('default-locking-mode'),
    }),
    define('more', noop, { label: '', isHideable: false }),
  ],
```